packages/discord.js/typings/index.d.ts

```diff
@@ -1642,6 +1642,46 @@ export class GuildMember extends Base {
   public valueOf(): string;
 }
 
+export interface UncachedGuildMember extends PartialTextBasedChannelFields<false> {}
+export class UncachedGuildMember extends Base {
+  private constructor(
+    client: Client<true>,
+    data: APIInteractionGuildMember | APIInteractionDataResolvedGuildMember,
+    guildId: Snowflake,
+  );
+  public avatar: string | null;
+  public get dmChannel(): DMChannel | null;
+  public get displayName(): string;
```

Comment:
i note infact from my previous comment that this is marked as always present but in the jsdoc it is not